frontend/src/app/main/data/plugins.cljs

```diff
@@ -8,13 +8,23 @@
   (:require
    [app.common.data.macros :as dm]
    [app.main.data.modal :as modal]
+   [app.main.data.notifications :as ntf]
    [app.main.store :as st]
    [app.plugins.register :as preg]
    [app.util.globals :as ug]
    [app.util.http :as http]
+   [app.util.i18n :as i18n :refer [tr]]
+   [app.util.time :as dt]
    [beicon.v2.core :as rx]
    [potok.v2.core :as ptk]))
 
+(defn save-plugin-permissions-peek
```

Comment:
I'm not very sure if `-peek` is the right suffix here, I think you may want something like `-pick`
And probably you should declare this function as private as it is used only from the same namespace

Reply:
This is like a "sneak peek" of the permissions. I think "-pick" doesn't make sense. I tough about "-cache" but that is misleading, it is not a cache...